ports/libplctag/portfile.cmake

```
@@ -0,0 +1,72 @@
+vcpkg_from_github(
+  OUT_SOURCE_PATH SOURCE_PATH
+  REPO libplctag/libplctag
+  REF "v${VERSION}"
+  SHA512 f540780642112483bd199995b327342bd2df2ec4a4d44c2356886ab94879aeb2da3e420760da2f00b5aee0d1278fc582f73cba8227b276787d21b18fa12a3add
+  HEAD_REF release
+  PATCHES
+	cmake.patch
+	sanitizer.patch
+)
+
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
```

This may need 
```suggestion
vcpkg_cmake_configure(
    SOURCE_PATH "${SOURCE_PATH}"
    DISABLE_PARALLEL_CONFIGURE
```
because 
https://github.com/libplctag/libplctag/blob/4793d92893906e80fcfc67d4c4ed87fd37d81df8/CMakeLists.txt#L327 
https://github.com/libplctag/libplctag/blob/4793d92893906e80fcfc67d4c4ed87fd37d81df8/CMakeLists.txt#L544 
may write to the source directory during configuration.

---

Good catch.  That may explain the spurious build failure I saw earlier on arm64_android.